lightning/src/ln/channel.rs

```diff
@@ -4114,6 +4127,26 @@ impl<Signer: Sign> Channel<Signer> {
 		}
 	}
 
+	/// Get forwarding information for the counterparty.
+	pub fn get_counterparty_forwarding_info(&self) -> Option<CounterpartyForwardingInfo> {
+		self.counterparty_forwarding_info.clone()
+	}
+
+	pub fn channel_update(&mut self, msg: &msgs::ChannelUpdate) -> Result<(), ChannelError> {
+		let usable_channel_value_msat = (self.channel_value_satoshis - self.counterparty_selected_channel_reserve_satoshis) * 1000;
+		if msg.contents.htlc_minimum_msat >  usable_channel_value_msat {
+			return Err(ChannelError::Close("Minimum htlc value is greater than channel value".to_string()));
+		}
+		self.counterparty_htlc_minimum_msat = msg.contents.htlc_minimum_msat;
```

Comment:
Isn't this *our* htlc_minimum? Its the value that the counterparty is announcing to the world as "the minimum HTLC value (in millisatoshi) that the channel peer will accept.", and I think *we're" the "channel peer".

Reply:
Oops, my bad. Hm, so the counterparty is unable to update the min htlc that they'll accept, then? Weird.